Week2/queryPackage.js

```diff
@@ -0,0 +1,90 @@
+
+const languagesInACountry = `with langs as
+                            (select country.Name, country.Code, countrylanguage.Language
+                                from new_world.country join new_world.countrylanguage
+                                on country.Code = countrylanguage.CountryCode)
+                            select Language from langs where Name = ?`;
+
+                            
+const capitalCityQuery = `with result as
+                        (select city.name as Capital, country.name as Country
+                            from new_world.country join new_world.city
+                            on country.Capital=city.ID)
+                        select Capital from result where Country = ?`
+
+
+const numberOfCitiesLangSpoken = `with result as
+                                (select countrylanguage.Language, countrylanguage.CountryCode, city.Name
+                                from new_world.countrylanguage join new_world.city
+                                on countrylanguage.CountryCode = city.CountryCode)
+                                select * from result where Language = ?`
+
+
+const languagesInARegion = `with result as
+                    (select country.Name, country.Region, country.Code, countrylanguage.Language
+                        from new_world.country join new_world.countrylanguage
+                        on country.Code = countrylanguage.CountryCode)
+                    select Language from result where Region = ?`
+
+
+const officialLanguagesInRegion = `with result as
```

Comment:
The `with result as` syntax is recommended when you want to reuse it in the subqueries. 
In your case, the `result` subquery is only used once in a single query. So it is not necessary to use that alias. The query could be written as 
```
                    select Language from (select country.Name, country.Region, country.Code, countrylanguage.Language
                        from new_world.country join new_world.countrylanguage
                        on country.Code = countrylanguage.CountryCode) where Region = ?`
```